Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -1277,7 +1279,15 @@ def fetch_incidents(client: Client,
         topdesk_incident['mirror_instance'] = demisto.integrationInstance()
         if float(last_fetch_datetime.timestamp()) < float(incident_created_time.timestamp()):
             labels = []
-            actions = client.list_actions(incident_id=topdesk_incident['id'], incident_number=None)
+            try:
+                actions = client.list_actions(incident_id=topdesk_incident['id'], incident_number=None)
+            except DemistoException as error:
+                demisto.debug(f'{error=}')
+                # make sure we catch only JSONDecodeError errors, in case it is a different exception, should be raised.
```

Comment:
can we change the exception that is thrown instead?

Reply:
no because its raised directly from `http_request` method in our base client which does a wrapping to it.

Reply:
see here: https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L8643